test/BUILD

```diff
@@ -1,13 +1,17 @@
 cc_test(
-    name = "tests",
-    srcs = ["hello_world_test.cc", "scsi_defs_test.cc", "nvme_defs_test.cc", "status_translator_test.cc" ],
-    deps = [
-        "//lib:hello_world_lib",
-        "//lib:scsi_defs_lib",
-        "//lib/util:optional_lib",
-        "//third_party/spdk_defs:nvme_defs_lib",
-        "//lib/translate:translator_lib",
-        "@googletest//:gtest_main",
-        "@com_google_absl//absl/base",
-    ],
+  name = "tests",
+  srcs = [
+    "scsi_defs_test.cc",
+    "nvme_defs_test.cc",
+    "status_translator_test.cc",
+    "lun_translator_test.cc"
```

Comment:
Would be good to put this in its own target. There's already a PR out that does so for master